kount/request.py

```diff
@@ -130,6 +130,11 @@ def __init__(self):
         self.params["SDK"] = "python"
         self.payment = None
         self.close_on_finish = None
+        card_solted = Khash.hash_payment_token(token="666666669")
```

Comment:
fixed in next commit. but in python good practices it's recommended to check with real example